### PR TITLE
Add an example of of custom `__repr__`

### PR DESCRIPTION
**Pull request by osmin625**
I add an example to the document to make it easier for beginners to understand the repr function.

----
:books: Documentation preview :books:: https://cpython-previews--112761.org.readthedocs.build/

### CODE REVIEW

File: Doc/library/functions.rst [hunk 1]
```diff
@@ -1569,6 +1569,16 @@ are always available.  They are listed here in alphabetical order.
    If :func:`sys.displayhook` is not accessible, this function will raise
    :exc:`RuntimeError`.
 
+   This class has a custom representation that can be evaluated.::
```
Comment (terryjreedy) [inline Doc/library/functions.rst]:
```suggestion
   This class has a custom representation that can be evaluated::
```

**Comment by terryjreedy** [comment]
Examples in this chapter are generally just a few lines.  Readers should experiment with usage, but the basic ideas should be in the tutorial.  Anyway, I am willing to merge this much.

**Comment by osmin625** [comment]
@terryjreedy  I actually added a minor example to gain experience contributing to open source as a Python-loving student. I would be thrilled if that commit was added, but I understand it's such a minor example that we don't have to add.
Thank you for leaving a comment on this pr.

**Comment by terryjreedy** [comment]
Both backports merged.

**Pull request merged**